test/index.js

```diff
@@ -66,6 +66,26 @@ it('should close instance when pressing Escape key', function() {
 	}
 });
 
+it('should NOT close instance when pressing Escape key if explicitClose option is set to TRUE', function() {
```

Comment:
@zjovic avoid negation in sentences, maybe something like `should leave dialog if explicitClose is true and Escape key is pressed`

Reply:
@niksy reformulated the sentences to avoid negation